Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -431,6 +442,17 @@
   - UplinkAmmo
 
+- type: listing
+  id: UplinkHighCapacityMagazinePistol
+  name: uplink-highcap-pistol-mag-name
+  description: uplink-highcap-pistol-mag-desc
+  icon: { sprite: /Textures/Objects/Weapons/Guns/Ammunition/Magazine/Pistol/pistol_high_capacity_mag.rsi, state: red }
+  productEntity: MagazinePistolHighCapacity
+  cost:
+    Telecrystal: 2
+  categories:
+  - UplinkAmmo
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)